app/controllers/twitch_live_streams_controller.rb

```diff
@@ -0,0 +1,12 @@
+class TwitchLiveStreamsController < ApplicationController
+  before_action :set_cache_control_headers
+
+  def show
+    @user = User.find_by!(username: params[:username].tr("@", "").downcase)
+    if @user.twitch_username.present?
+      render :show
+    else
+      render :no_twitch
```

Comment:
The current template for this is VERY minimal. Maybe it should redirect somewhere else instead?